src/main/java/com/github/kaklakariada/fritzbox/model/homeautomation/ColorControl.java

```diff
@@ -38,6 +38,18 @@
     @Attribute(name = "current_mode", required = false)
     private String current_mode;
 
+    @Attribute(name = "fullcolorsupport", required = false)
+    private String fullcolorsupport;
+
+    @Attribute(name = "mapped", required = false)
+    private String mapped;
+
+    @Element(name = "unmapped_hue", required = false)
+    private String unmapped_hue;
```

Comment:
Please rename the field to use camelCase. Same for the other attributes and getters.

Reply:
done